### DISCUSSION TITLE
[Q] How would I ignore excel error in ExcelJS?

### DISCUSSION THREAD

**Discussion by PyaeNaing** [exceljs/exceljs#2369] [Q&A]
How do I ignore excel error or disable background error checking for the entire workbook in ExcelJS.

Currently I am trying to ignore "Number stored as text" error.

I do not want to store number as number I want to keep it as text, and just want to ignore the error so it does not show up in excel.

I've read the exceljs documentation and tried to search answer but seems this has not been answered, appreciate any help.

**Comment by fullstackzach** [comment]
I have the same desire to ignore errors - I have IDs I want to express in excel as a string, and they are passed that way. Some IDs may be in numerical format (e.g. 1234) and excel shows the "Number stored as text" error.

**Comment by shahrukhappit** [comment]
Same desire me too. let us know if there any possibilities. Thanks

**Comment by mustafa-alamir** [comment]
Same desire me too. let us know if there any possibilities. Thanks

**Comment by aashaysinghai** [comment]
Same desire me too. let us know if there any possibilities. Thanks

**Comment by DomenicoColandrea86** [comment]
I am also having this issue.

Any update?

[This](https://github.com/exceljs/exceljs/discussions/1910) looks like a good suggestion.
